ModularTegustation/tegu_items/refinery/crates/_crate.dm

```diff
@@ -29,11 +29,22 @@
 	var/veryrarechance
 	var/cosmeticloot = list()
 	var/cosmeticchance = 0 //These do not count on the total odds of a crate
+	var/repmodifier = 0
 
 /obj/structure/lootcrate/Initialize()
 	. = ..()
 	if(SSmaptype.maptype in SSmaptype.citymaps)	//Also can't drag it out to open it. Open it on spot, bitch
 		anchored = TRUE
+	for(var/upgradecheck in GLOB.jcorp_upgrades)
+		switch(upgradecheck)
+			if("Gacha Chance 4")
+				repmodifier += 3
+			if("Gacha Chance 3")
+				repmodifier += 3
+			if("Gacha Chance 2")
+				repmodifier += 2
+			if("Gacha Chance 1")
+				repmodifier += 2
```

Comment:
are these chances right?
feels a bit weird that level 3 is the same as 4, and 1 the same as 2